CONTRIBUTING.md

```diff
@@ -32,11 +32,12 @@ Every participant(s) is required to fork this repository and make changes on the
     # To add all new files to branch Branch_Name
     $ git add .
     ```
-To commit give a descriptive message for the convenience of reveiwer by:
+- Commit messages should follow a [certain guideline](https://udacity.github.io/git-styleguide/).To commit give a descriptive message for the convenience of reveiwer by:
     ```
     # This message get associated with all files you have changed
     $ git commit -m 'message'
```

Comment:
Change it according to the style guide. It should be 
    $ git commit -m "message header" -m "message body" -m "message footer"